assets/less/elementor-frontend-forms.less

```
@@ -0,0 +1,254 @@
+	&.wpuf-elementor-hide-labels .wpuf-label {
+		display: none !important;
+	}
+
+	// Hide error messages option
+	&.wpuf-elementor-hide-errors .wpuf-error,
+	&.wpuf-elementor-hide-errors .error {
+		display: none;
+	}
+
+	// Hide placeholder option
+	&.wpuf-elementor-hide-placeholder input::-webkit-input-placeholder,
```

Stylelint (17.2.0)

[error] Unexpected vendor-prefixed selector "::-webkit-input-placeholder" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)